src/pages/api/github/pr.ts

```diff
@@ -421,6 +421,7 @@ export default async function handler(
     prUrl,
     ghSourcePath,
     ghBranchUrl,
+    ...otherMetaData
```

Comment:
nit: metadata is one word. We already have the `formatDataForMetadata` in which you can see that.
```suggestion
    ...otherMetadata
```

also, if at this point we are talking about arbitary metadata, maybe it can be called `arbitaryMetadata`